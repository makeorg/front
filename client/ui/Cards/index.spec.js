import React from 'react';
import renderer from 'react-test-renderer';
import { CardStyle } from './index';

describe('CardStyle', () => {
  it('snapshot by default', () => {
    const component = renderer.create(<CardStyle />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
