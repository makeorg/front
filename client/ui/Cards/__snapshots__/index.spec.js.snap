// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardStyle snapshot by default 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  max-width: 71.25rem;
  padding: 0.9375rem;
  background-color: rgb(255,255,255);
  box-shadow: 0 1px 1px 0 rgba(0,0,0,0.5);
}

<section
  className="c0"
/>
`;
